zds/member/api/serializers.py

```diff
@@ -17,10 +17,11 @@ class UserListSerializer(serializers.ModelSerializer):
     serializers.
     """
     avatar_url = serializers.CharField(source='profile.get_avatar_url')
+    link = serializers.CharField(source='get_absolute_url')
```

Comment:
C'est toujours étrange qu'une api REST ne donne pas l'url vers la 
ressource. Donc pour moi, je suis d'accord avec @sandhose

Le 28/03/2016 11:18, Gérard Paligot a écrit :
>
> In zds/member/api/serializers.py 
> <https://github.com/zestedesavoir/zds-site/pull/3265#discussion_r57556945>:
>
> > @@ -17,10 +17,11 @@ class UserListSerializer(serializers.ModelSerializer):
> >      serializers.
> >      """
> >      avatar_url = serializers.CharField(source='profile.get_avatar_url')
> > +    link = serializers.CharField(source='get_absolute_url')
>
> Je n'aime toujours pas ça. Je laisse @SpaceFox 
> <https://github.com/SpaceFox> tranché si on veut oui ou non rajouter 
> cette information quand on liste les utilisateurs via l'API.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3265/files/f896364cc5fa324a343a91e6861e7050ce7e3ae6#r57556945>
>